### ISSUE TITLE
TF2 tutorial bots are broken

### ISSUE THREAD

**Issue by Pinsplash** [ValveSoftware/Source-1-Games#3345]
Bots in the map tr_target (and probably tr_dustbowl as well) are broken and almost never spawn, rendering the tutorials impossible to complete.

**Comment by AshThe9thSurvivor** [comment]
You are right, there comes a time when the bots stop appearing and the tutorial never ends, all the new ones who enter do not know what else to do since they get trapped without knowing what continues.
